nncf/config/schema.py

```diff
@@ -370,11 +370,31 @@ def with_attributes(schema: Dict, **kwargs) -> Dict:
     "additionalProperties": False,
 }
 
+ACCURACY_AWARE_SCHEMA = {
+    "maximal_accuracy_degradation": with_attributes(_NUMBER,
+                                                    description="Maximally allowed accuracy degradation"
+                                                                " of the model"),
+    "initial_training_phase_epochs": with_attributes(_NUMBER,
+                                                     description="Number of epochs to tune during the initial"
+                                                                 "training phase"),
+    "initial_compression_rate_step": with_attributes(_NUMBER,
+                                                      description="initial_compression_rate_step"),
```

Comment:
Could you change the description to something more readable and understandable instead of "initial_compression_rate_step"? And below too.

Reply:
done